module/journal-sync.js

```diff
@@ -320,7 +372,7 @@ async function exportFolder(folder, parentPath) {
     let folderPath = (parentPath + '/' + folder.data.name).replace("//", "/").trim();
 
     // Create folder directory on server. 
-    FS.createDirectory("data", folderPath)
+    FilePicker.createDirectory(markdownPathOptions.activeSource, folderPath)
```

Comment:
folderPath is a combination of parent and folder. Looks like there is no check for parent folder to see if it exists. Create directory could just be duplicated for a parent check.

Reply:
Fair enough, I realize I missed that on my internal version yesterday night as well as I was making a poor assumption on world's existing from a workflow view. MVP based on tightly scoped requirements? ;-) I'll fix it.

Reply:
Side note, in testing I don't see that the catch on FilePicker.createDirectory is every really called. It looks like FilePicker.createDirectory is fully non-destructive and non-returning in the case of existence (checked for by EEXIST)... just a note.

Reply:
Yea, I think I was just trying to catch the exception in a nicer way compared to another error I was seeing at the time.